nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/pom.xml

```diff
@@ -77,6 +76,19 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-serialization-service-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-record-utils</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
```

Comment:
`commons-csv` dependency is no longer needed. Please remove it.

Reply:
This is still used:

import org.apache.commons.csv.CSVFormat;
import org.apache.commons.csv.CSVPrinter;

Reply:
I think by having a dependency to nifi-standard-record-utils, we don't have to add commons-csv explicitly here. I'm fine with keeping it, but I prefer having less dependencies.

Reply:
Your correct. I've removed it.